core-bundle/src/Resources/config/services.yml

```diff
@@ -108,6 +108,14 @@ services:
         #    message: Using the "%alias_id%" service ID has been deprecated and will no longer work in Contao 5.0. Please use "Contao\CoreBundle\Controller\BackendCsvImportController" instead.
         public: true
 
+    Contao\CoreBundle\Controller\ContentElement\MarkdownController:
+        factory: ['Contao\CoreBundle\Controller\ContentElement\MarkdownController', 'createController']
+        arguments:
+            - '@request_stack'
+            - '%kernel.project'
```

Comment:
This parameter is not needed I think?

Reply:
Yes, just removed that in my latest CS commit :)